apache/doris
# [wg](chore) rename workload group memory property names

### What problem does this PR solve?

This is a property behaviour change. Since spill disk is not many used, we can change its name.

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 

- Behavior changed:

- Does this need documentation?

### Check List (For Reviewer who merge this PR)


Mannual test steps:
1. With old FE and BE, modify properties to new value:
```
alter workload group normal properties("spill_threshold_low_watermark"="55%");
alter workload group normal properties("spill_threshold_high_watermark"="85%");
```
2. Update BE to the modified code, check that values are the update values in step 1.
3. Modify properties values again:
```
alter workload group normal properties("spill_threshold_low_watermark"="60%");
alter workload group normal properties("spill_threshold_high_watermark"="90%");
```
4. Update FE to the modified code,  check that values are the update values in step 3.
5. Modify properties values again:
```
alter workload group normal properties("memory_low_watermark"="65%"); 
alter workload group normal properties("memory_high_watermark"="95%"); 
```
